lte/gateway/python/magma/pipelined/tests/app/subscriber.py

```diff
@@ -14,18 +14,18 @@
 import logging
 import time
 from collections import namedtuple
-from concurrent.futures import Future
 
 import abc
 import grpc
 from lte.protos.pipelined_pb2 import ActivateFlowsRequest, \
     DeactivateFlowsRequest
-from magma.pipelined.policy_converters import convert_ipv4_str_to_ip_proto
 from ryu.lib import hub
 
 from magma.subscriberdb.sid import SIDUtils
+from magma.pipelined.policy_converters import convert_ip_str_to_ip_proto
 
-SubContextConfig = namedtuple('ContextConfig', ['imsi', 'ip', 'ambr', 'table_id'])
+SubContextConfig = namedtuple('ContextConfig', ['imsi', 'ip', 'ambr',
```

Comment:
Initially I wanted to convert the ip string to IPAddress, but the `SubContextConfig` is used all over the codebase so I just internally use the string and determine ipaddress type

Reply:
I am not sure what is plan, are we going to continue to use sting here and in API? is that is case we shld get rid of IPAddress struct, since it requires unnecessary encoding and decoding of IP address.

Reply:
This SubContextConfig is only used in testing, having the ip_proto makes the overall system a bit easier for actual rule install. And long term we're getting rid of the ip anyways and moving to tunnel

Reply:
ok, I hope we get rid of ip-address sooner :)